src/net/sourceforge/kolmafia/AdventureResult.java

```diff
@@ -1504,4 +1505,44 @@ public static WildcardResult getInstance(String text) {
       return null;
     }
   }
+
+  public static AdventureResult parseEffectString(String effectString) {
```

Comment:
How does this parse `Educated (Kinda)`? Can it work like item instead?

Reply:
Yeah good idea. This was an issue with the old function that surfaced with `filet of tangy gnat (fotlief)`